addons/nvidia-bootstrap/image/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Simple early detection of nvidia card
+grep 10de102d /proc/bus/pci/devices || exit 0
```

Comment:
It was the K80 (?) that was on the p2.xlarge that I tested with.  The goal is not to hold up the installation of the master while we download nvidia drivers for a node which probably doesn't have a GPU.